palindrome.c

```diff
@@ -0,0 +1,36 @@
+/* Jared Wasserman -- palindrome.c */
+/*This program takes a string of input and returns if it is a palindrome or not.*/
+#include <stdio.h>
```

Comment:
Perfect job! You're literally only the second person whose program was able to handle capitals! 💯